### DISCUSSION TITLE
Neotree bug has deleted my entire project twice this week

### DISCUSSION THREAD

**Discussion by DanielPower** [nvim-neo-tree/neo-tree.nvim#1090] [General]
Since upgrading to Neotree 3.x, I've been periodically experiencing an issue where Neotree always thinks the root node is selected regardless of where my cursor is.

I do not know how to get into this state, or how to consistently reproduce. But once in this state, pressing enter anywhere in the Neotree window will collapse the root note. And much more significantly, pressing d will prompt you to delete the root node.

This has resulted in two events where I have deleted my entire project folder (including the .git directory), when I attempted to delete a single file.

I am not opening this as an issue because I do not have specific reproduction steps, and I do not have a minimal config to reproduce with. However I want to warn users that this is a potential, and that I will no longer be using this plugin as a result.

My config at the time:
https://github.com/DanielPower/dotfiles/tree/5a550150ef0925e4f26478e06f956f1b410baa41/.config/nvim

The file containing my Neotree config:
https://github.com/DanielPower/dotfiles/blob/5a550150ef0925e4f26478e06f956f1b410baa41/.config/nvim/lua/plugins/neotree.lua

**Comment by rafo** [comment]
I just had a similar incident after moving files. A copy of the directory I moved to appeared. After I deleted the copy, the original directory was also gone.
